j2cl-tasks/src/main/java/com/vertispan/j2cl/tools/Closure.java

```diff
@@ -112,6 +114,15 @@ public boolean compile(
             jscompArgs.add(extern);
         }
 
+        translationsFile.ifPresent(file -> {
+            if(compilationLevel.equals(CompilationLevel.ADVANCED_OPTIMIZATIONS)) {
```

Comment:
stupid question perhaps, but are we sure this doesnt work with other modes?

...with that said, I dont think WHITESPACE_ONLY works at all right now, and I think SIMPLE is broken on windows

Reply:
@niloc132 in this case we should create a check list of available modes and test PR againt them. I think, we can allow translationFiles to work with ADVANCED mode only.

Reply:
Sounds good, defer it to another issue?

Reply:
yes, i think so